backend/pom.xml

```diff
@@ -72,6 +72,11 @@
             <artifactId>springfox-swagger-ui</artifactId>
             <version>2.9.2</version>
         </dependency>
+        <dependency>
+            <groupId>com.opencsv</groupId>
+            <artifactId>opencsv</artifactId>
+            <version>4.1</version>
```

Comment:
Szerintem az egy jó szokás, hogy ha használtok valamilyen lib-et, akkor megnézitek, hogy milyen verziók érhetők el, és nem használjátok feltétlenül azt, amit a talált kódban használnak. Szerintem ez kb max 15 perc további nézelődést igényel: https://mvnrepository.com/artifact/com.opencsv/opencsv
Itt látható, hogy a választott lib 2017-es, amivel 4 év bugfixelését és fejlesztését hagyjátok ki. Természetesen lehetséges, hogy valami miatt pont valamelyik régi lib-et kell használni, de talán ez az eset nem igaz erre.

Reply:
Továbbá hozzáadtok egy rakást file-t alul a resource-hoz ami erre biztosan nem való. Adjatok hozzá egy endpointot, amin keresztül fel lehet tölteni file-okat, amiker a rendszer feldolgoz és eltárolja őket (vagy az adataikat), ami alapján már még egyszer ugyanez a kontent nem kerül importálásra.